decidim-core/app/views/decidim/messaging/conversations/_message.html.erb

```diff
@@ -1,14 +1,13 @@
-<article class="message">
-  <div class="message__header">
-    <div class="card__author author-data author-data--small">
-      <%= cell "decidim/author", present(message.sender), context: { extra_classes: ["author-data--small"] } %>
-
-      <time datetime="<%= message.created_at.utc %>">
-        <%= message.friendly_created_at %>
-      </time>
-    </div>
+<div class="conversation-chat<%= " conversation-chat--offset" if (current_user.id == message.sender.id) %>">
```

Comment:
I see you changed the class "message" to "conversation-chat". "message" wouldn't work there anymore? Do you know if this class is unused or if it is used properly across other files? Because if it's not used anymore, it would be good to remove it from the css files

Reply:
You're right! I've looked for classes "message", "message_header" and "message_content" in other files without any match, so I deleted them from _messages.scss file

Reply:
Okay! But one thing, you deleted `.message__header` and `.message__content`, but `.message` is still there, do you accidentally skipped it?

Reply:
Yes, you're right! I left it that way because the .add-message dependency with .message:

![image](https://user-images.githubusercontent.com/9702463/80383528-f635f700-88a3-11ea-8f63-b5afe73d026e.png)

but I didn't realized that it's cleaner in that other way:

![image](https://user-images.githubusercontent.com/9702463/80383419-d6063800-88a3-11ea-83c8-622e3e976737.png)

Reply:
All right, good job!